src/dstack/_internal/cli/main.py

```diff
@@ -6,11 +6,12 @@
 from dstack._internal.cli.commands.gateway import GatewayCommand
 from dstack._internal.cli.commands.init import InitCommand
 from dstack._internal.cli.commands.logs import LogsCommand
+from dstack._internal.cli.commands.pool import PoolCommand
 from dstack._internal.cli.commands.ps import PsCommand
 from dstack._internal.cli.commands.run import RunCommand
 from dstack._internal.cli.commands.server import ServerCommand
 from dstack._internal.cli.commands.stop import StopCommand
-from dstack._internal.cli.utils.common import colors, console
+from dstack._internal.cli.utils.common import _colors, console
```

Comment:
Change _colors to colors or COLORS. Leading underscore means the var is private to the module.

Reply:
I've changed it recently to private because we were tempted to use it for rich formatting with the theme-customized `console`. This import is the only place, where `_colors` is being used "externally"

Reply:
I'm going to leave this part unchanged.